app/views/controllers/projects/violations/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<%
+  add_page_title(:violations_edit_title.t)
+  @container = :wide
+%>
+
+<h4>
+  <%= :PROJECT.l + ": " + link_to_object(@project).t %>
```

Comment:
@nimmolo:
`link_to_object` doesn't work -- there's no href.  I think I saw a similar issue raised by @nathan. 
Here the line renders as:
```html
 <a>First Fantastic Fungal Friends Foray Festival!</a>
```
In this PR, I'll probably replace the link with the project name. (I no longer need the link; Submit/Show Project button makes the link redundant.)

Reply:
Conversation resolved. I removed link_to_object, used `tag.h4` instead of html